doc/en/backwards-compatibility.rst

```diff
@@ -83,6 +83,7 @@ Released pytest versions support all Python versions that are actively maintaine
 ==============  ===================
 pytest version  min. Python version
 ==============  ===================
+9.0+            3.9+
```

Comment:
Not sure we will make 9.0 the next release, dependency changes like this are usually not a reason to bump the major.

```suggestion
8.4+            3.9+
```